lib/net/pool.js

```diff
@@ -4214,8 +4322,14 @@ class PoolOptions {
     this.port = this.network.port;
     this.publicHost = '0.0.0.0';
     this.publicPort = this.network.port;
+    this.brontideHost = '0.0.0.0';
+    this.brontidePort = this.network.brontidePort;
+    this.brontidePublicHost = '0.0.0.0';
+    this.brontidePublicPort = this.network.brontidePort;
     this.maxOutbound = 8;
     this.maxInbound = 8;
+    this.brontideMaxInbound = 2;
+    this.brontideMaxOutbound = 0;
     this.createSocket = this._createSocket.bind(this);
```

Comment:
The default values for inbound and outbound connections are here. I set the max brontide inbound to 2 as to not add a lot of extra bandwidth usage. The max outbound is set to 0 as to not infinitely loop on trying to connect to outbound peers using brontide. If `hnsd` only supports brontide at first, then bumping the `brontideMaxInbound` could be helpful.

Reply:
We should add a `TODO` about bumping `brontideMaxInbound` if `hnsd` is brontide-only.